chpl/chpl-service/src/main/java/gov/healthit/chpl/manager/SchedulerManager.java

```diff
@@ -159,6 +162,35 @@ public void deleteTrigger(final String triggerGroup, final String triggerName)
         return triggers;
     }
 
+    @PreAuthorize("@permissions.hasAccess(T(gov.healthit.chpl.permissions.Permissions).SCHEDULER, "
+            + "T(gov.healthit.chpl.permissions.domains.SchedulerDomainPermissions).GET_ALL_TRIGGERS)")
+    public List<ScheduledSystemJob> getScheduledSystemJobsForUser() throws SchedulerException {
+        List<ScheduledSystemJob> ssJobs = new ArrayList<ScheduledSystemJob>();
+        Scheduler scheduler = getScheduler();
+        for (String group : scheduler.getTriggerGroupNames()) {
+            for (TriggerKey triggerKey : scheduler.getTriggerKeys(groupEquals(group))) {
+                if (scheduler.getTrigger(triggerKey).getJobKey().getGroup().equalsIgnoreCase(SYSTEM_JOBS_KEY)) {
+                    if (doesUserHavePermissionToTrigger(scheduler.getTrigger(triggerKey))) {
+                        Trigger curTrigger = getScheduler().getTrigger(triggerKey);
+                        String jobName = curTrigger.getKey().getName();
+                        JobDetail jobDetail = getScheduler().getJobDetail(getScheduler().getTrigger(triggerKey).getJobKey());
+                        String jobDescription = jobDetail.getDescription();
+                        Date nextRunDate = curTrigger.getNextFireTime();
+                        if (curTrigger instanceof CronTrigger) {
+                            ssJobs.add(new ScheduledSystemJob(jobName, jobDescription, nextRunDate,
+                                    TriggerSchedule.REPEATABLE));
+                        } else if (curTrigger instanceof SimpleTrigger) {
+                            jobName = curTrigger.getJobKey().getName();
+                            ssJobs.add(new ScheduledSystemJob(jobName, jobDescription, nextRunDate,
+                                    TriggerSchedule.ONE_TIME));
+                        }
+                    }
+                }
+            }
+        }
```

Comment:
Not going to hold this up, but we should be trying to organize this a little more functionally using `streams`.

Reply:
I will keep this on my radar and maybe I can update it in the middle of other work if time allows. But, it may not make this PR.